nav2_controller/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ project(nav2_controller)
 find_package(ament_cmake REQUIRED)
 find_package(nav2_core REQUIRED)
 find_package(nav2_common REQUIRED)
+find_package(angles REQUIRED)
```

Comment:
This must be added to the package.xml as well

Reply:
Done